### ISSUE TITLE
Screenreader support

### ISSUE THREAD

**Issue by tehapo** [vaadin/vaadin-date-picker#70]
A visually impaired user should be able to select a specific date with the help of a screenreader, using the keyboard.

After #196, the user should still be able to navigate the calendar list and to choose a specific date using that instead of just typing in a valid date in the input field.

Should be validated at least on Windows and macOS.

#### Phase 1 DoD
- The team should be able to select a specific date with the help of a screenreader, using the keyboard, with the following:
 - macOS VoiceOver on Chrome, Firefox and Safari
 - JAWS (free trial) and NVDA on Windows
 - iOS VoiceOver
 - Android Talkback

#### Phase 2 DoD
- A visually impaired outside user should be able to select a specific date (perhaps ask on the previously mentioned Reddit thread for help)
 - We assume this is just a validation for the work, and doesn’t require much extra work from us

**Comment by Saulis** [comment]
https://www.w3.org/TR/wai-aria-practices-1.1/#datepicker

**Comment by platosha** [comment]
See also: #256
